requirements.txt

```diff
@@ -1,5 +0,0 @@
-musica==0.6.1.dev0
-pandas
-pipx
-pytest
-twine
```

Comment:
was this file deleted on purpose? it seems to still be referenced in some of the GH Actions, and is causing them to fail

Reply:
I don't know how requirements.txt disappeared from music-box-carls-main01. In any case, I have restored the requirements file and updated the musica version to **0.7.0**.